scripts/load_countries_cities.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+
+def run():
+    pwd = input("Write the countries cities data json file path from root repo: ")
+    pwd = "./scripts/countries_cities_data.json" if pwd == "" else pwd
+
+    with open(pwd, "r") as f:
```

Comment:
Could you give a more meaningful name for f?
like file? json_file? data_file? or maybe coutry_city_data?